README.md

```diff
@@ -14,36 +14,68 @@ to your list of dependencies in `mix.exs`:
 ```elixir
 def deps do
   [
-    {:redis_mutex, "~> 0.4.0"}
+    {:redis_mutex, "~> 1.0"}
   ]
 end
 ```
 
 
 ## Usage
 
-1. Set the `redis_url` in your `config.exs`
+### Adding `RedisMutex` to your application's supervision tree
+
+Add `RedisMutex` to your application's supervision tree.
+
+```elixir
+  @impl Application
+  def start(_type, _args) do
+    children = [other_children() | RedisMutex]
+    Supervisor.start_link(children, strategy: :one_for_one, name: MyApp.Supervisor)
+  end
+```
+
+If you do not have an instance of redis running when you run your test suite, exclude `RedisMutex` from your
+application's supervision tree in test.
+
+```elixir
+  @impl Application
+  def start(_type, _args) do
+    children =
+      if Mix.env() == :test do
```

Comment:
Not to hold our examples to too high a standard, but projects using releases will not have Mix available here.

Reply:
Is there a different way for people to keep things out of the supervision tree in test when using releases?